assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "grey water"
+          },
+          "question": {
+            "en": "Can you dispose of grey water here?"
+          },
+          "freeform": {
+            "key": "sanitary_dump_station:grey_water"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=yes"
+                ]
+              },
+              "then": {
+                "en": "You can dispose of grey water here"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of gray water here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "chemical toilet"
+          },
+          "question": {
+            "en": "Can you dispose of chemical toilet waste here?"
+          },
+          "freeform": {
+            "key": "sanitary_dump_station:chemical_toilet"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:chemical_toilet=yes"
+                ]
+              },
+              "then": {
+                "en": "You can dispose of chemical toilet waste here"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:chemical_toilet=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of chemical toilet waste here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "access tag"
+          },
+          "question": {
+            "en": "Who can use this dump station?"
+          },
+          "freeform": {
+            "key": "access"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "access=network"
+                ]
+              },
+              "then": {
+                "en": "You need a network key/code to use this"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=customers"
+                ]
+              },
+              "then": {
+                "en": "You need to be a customer of camping/campersite to use this place"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=public"
+                ]
+              },
+              "then": {
+                "en": "Anyone can use this dump station"
+              },
+              "hideInAnswer": false
```

Comment:
False is the default - you can omit this; but it is not wrong per se.

BTW: isn't `access=yes` the preferred tag over `access=public`? Then maybe this one should be hidden and the one below be shown.

Reply:
oh dear. Of course. Fixed.